### PR TITLE
Update MetaDataModelType instances

### PR DESCRIPTION
**Pull request by lzehl**
adding preliminary definitions; correcting names; adding terms

### CODE REVIEW

File: terminologies/metaDataModelType/commonDataElements.jsonld [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "@id": "https://openminds.ebrains.eu/controlledTerms/metaDataModelType/commonDataElements",
   "@type": "https://openminds.ebrains.eu/controlledTerms/MetaDataModelType",
   "name": "common data elements",
-  "definition": null,
+  "definition": "Common Data Element (CDE) define standardized key terms or concepts for describing clinical research or studies in order to enhance data quality and cross validations or visualizations.",
```
Comment (UlrikeS91) [inline terminologies/metaDataModelType/commonDataElements.jsonld]: "Common Data Element**S** (CDEs) define..." (typo?)

File: terminologies/metaDataModelType/relationalMetadataModel.jsonld [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "https://openminds.ebrains.eu/controlledTerms/metaDataModelType/relationalMetadataModel",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MetaDataModelType",
+  "name": "relational metadata model",
+  "definition": "A graph metadata model defines a set of tabular metadata schemas (including their relations) as architectural base of a relational database for describing the products represented in that database.",
```
Comment (UlrikeS91) [inline terminologies/metaDataModelType/relationalMetadataModel.jsonld]: "A **relational** metadata model ..." (copy-paste-error)

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]
See comments. Just some typos.

File: terminologies/metaDataModelType/commonDataElements.jsonld (hunk 1 above)
Comment (lzehl) [reply terminologies/metaDataModelType/commonDataElements.jsonld]: changed it to Common Data Element (CDE) literature is a bit confusing on the matter... 

sometimes only Common Data Element with abbreviation CDE is used, sometimes it's fully written as Common Data Elements, but still abbreviated as CDE, and sometimes it's fully written as Common Data Elements and abbreviated as CDEs... (I suggest to stick with the fully singular form for now; what do you think?)

File: terminologies/metaDataModelType/relationalMetadataModel.jsonld (hunk 2 above)
Comment (lzehl) [reply terminologies/metaDataModelType/relationalMetadataModel.jsonld]: yep, thanks for noticing. I've updated the description.

**Comment by lzehl** [comment]
@UlrikeS91 ready for the second check :slightly_smiling_face:

**Review by UlrikeS91 — APPROVED** [review]

File: terminologies/metaDataModelType/commonDataElements.jsonld (hunk 1 above)
Comment (UlrikeS91) [reply terminologies/metaDataModelType/commonDataElements.jsonld]: Okay, but do you want to also change filename, `@id` and name then? All have the plural s at the moment.

Comment (UlrikeS91) [reply terminologies/metaDataModelType/commonDataElements.jsonld]: I am approving it either way. Just want to make sure you want this inconsistency. So, I'll wait with the merging until you decided :)

Comment (lzehl) [reply terminologies/metaDataModelType/commonDataElements.jsonld]: mh... I thought I did. at least for the `@id`. must have not saved it or just forgotten about it while thinking it :see_no_evil: I'll change it right away

Comment (lzehl) [reply terminologies/metaDataModelType/commonDataElements.jsonld]: I forgot the filename and changed it now. `@id` was already changed.

**Comment by lzehl** [comment]
@UlrikeS91 if I did not forget something else :see_no_evil: you can merge now I think

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**